backend/src/main/java/gov/cdc/usds/simplereport/config/DataLoaders.java

```diff
@@ -0,0 +1,64 @@
+package gov.cdc.usds.simplereport.config;
```

Comment:
Hmmm. Well, I think as it stands right now it's a `service`, but pending the refactor below where the individual DataLoaders are Autowired into a map, I think it will go back to being a `config` that just registers the loaders. So I'm going to leave it alone.

Reply:
This file was replaced with a `DataLoaderRegistryConfiguration` that just news up a `DataLoaderRegistry` to resolve the Bean. The individual DataLoaders now just self-register with the singleton Registry.